org.eclipse.tm4e.language_pack/plugin.xml

```diff
@@ -1320,6 +1320,7 @@
   <extension point="org.eclipse.tm4e.registry.grammars">
     <grammar scopeName="text.xml" path="syntaxes/xml/xml.tmLanguage.json" />
     <scopeNameContentTypeBinding scopeName="text.xml" contentTypeId="org.eclipse.tm4e.language_pack.xml" />
+    <scopeNameContentTypeBinding scopeName="text.xml" contentTypeId="org.eclipse.core.runtime.xml"  />
```

Comment:
This section is auto-generated meaning next time we run the grammar updater it will be gone.

Can you instead add the following snippet somewhere before the `<!-- START-GENERATED -->` marker in the plugin.xml?
```xml
  <!-- ======================================== -->
  <!-- XML                                      -->
  <!-- ======================================== -->
  <extension point="org.eclipse.tm4e.registry.grammars">
    <scopeNameContentTypeBinding scopeName="text.xml" contentTypeId="org.eclipse.core.runtime.xml"  />
  </extension>
```

Reply:
Thanks, was not aware about that. See updated commit.